buildpacks/_shared/go.mod

```diff
@@ -0,0 +1,44 @@
+module kn-fn/shared
+
+go 1.19
+
+require (
+	github.com/paketo-buildpacks/libpak v1.61.0
+	gopkg.in/yaml.v3 v3.0.1
+	knative.dev/kn-plugin-func v0.19.0
+	knative.dev/pkg v0.0.0-20210902173607-844a6bc45596
+)
+
+require (
```

Comment:
Just doublechecking that a `go mod tidy` was run on all of your `go.mod`'s?

Reply:
Good suggestion, done!